src/main/java/fruitshop/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,27 @@
+package fruitshop.service.impl;
+
+import fruitshop.model.FruitTransaction;
+import fruitshop.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    public static final int TYPE = 0;
+    public static final int FRUIT = 1;
+    public static final int QUANTITY = 2;
+    public static final String COMA = ",";
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
```

Comment:
It's a good practice to validate input data. Consider adding a check for null or empty `inputReport` to avoid `NullPointerException` or unexpected behavior.